src/gauges/bartio/defaultGaugeList.json

```diff
@@ -188,6 +188,15 @@
         "0x9F43e40093327697Ff92d4b353D0A7B88b0DbBb6",
         "0x7507c1dc16935B82698e4C63f2746A2fCf994dF8"
       ]
+    },
+    {
+      "beraRewardsVault": "0xC4f202b367FCB32F16305bd445E6AB4b445cb738",
+      "lpTokenAddress": "0xb18CEC4160E5310e79AEf0d06a8df223A35599cD",
+      "mintUrl": "https://terplayer.org/terp-stake/v2",
+      "name": "tTERP Staking",
+      "protocol": "terplayer stake",
```

Comment:
```suggestion
      "name": "tTERP",
      "protocol": "tterp",
```

Reply:
same comment as above, I would suggest "terplayer" as the protocol id, just make sure this protocol field matches the protocol id given whatever value you end up with.

Reply:
Okay